.github/workflows/backend.yml

```diff
@@ -129,7 +142,7 @@ jobs:
   # test runs at all times to give them an early warning sign is something is broken.
   be-tests-java-11-ee-pre-check:
     needs: files-changed
-    if: github.event.pull_request.draft == true && needs.files-changed.outputs.backend_all == 'true'
+    if: github.event.pull_request.draft == false && (needs.files-changed.outputs.backend_all == 'true' || needs.files-changed.outputs.static_viz == 'true')
```

Comment:
Why would you change this to NOT run when a PR is in a draft mode?
This is a pre-check, a sanity check for people who are kind enough to keep their PRs in a draft mode as to not run all other workflows.

Reply:
Good catch. This wasn't intentional, I'll fix it 👍